sass/directives/_boxes.scss

```diff
@@ -10,3 +9,23 @@
   padding: $base-padding;
   background-color: $panel-primary;
 }
+
+// Accordion Panels
+@mixin accordion__panel {
```

Comment:
@arelia @ElliottAYoung For future reference, let's use SASS's placeholder selectors for SASS functions that don't accept any parameters. The code that SASS generates is infinitely more efficient (with a mixin, it duplicates the mixin's attributes for every element that inherits from it; with a placeholder selector, it is only produced once). This also means they can also be responsibly mixed-into mixins.
See an example of these in use in this repo: https://github.com/cb-talent-development/employer-style-base/blob/master/sass/directives/_flashes.scss#L14
See this as a handy intro: http://thesassway.com/intermediate/understanding-placeholder-selectors

Additionally, placeholder selectors can be used to great effect in the form of extend-only selectors. These can be used to generate very clean code, and still introduce some dynamicism. See: http://www.sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholders

Since this pattern isn't adhered to everywhere in `style-base`, I'll go through myself and clean things up. Just an important note for future ref. Thanks!